sli/typechk.cc

```diff
@@ -255,7 +252,7 @@ TypeTrie::insert_move( const TypeArray& a, Token& f )
   /* Error conditions:
      1. If pos->next!=NULL, the parameter list overlaps with
```

Comment:
@JanVogelsang Just one little NULL left ;)
```suggestion
     1. If pos->next != nullptr, the parameter list overlaps with
```